bugbug/db.py

```diff
@@ -115,13 +115,21 @@ def upload(path):
 
 
 def last_modified(path):
```

Comment:
I know the issue title was saying to return None in case the schema was different, but let's actually implement the same behavior as when Last-Modified is not available.
So, to summarize, raise the Exception("Last-Modified is not available") instead of returning None.

Also, it would be good to create a custom exception, but we can do that in a follow-up PR.

Reply:
I filed #2459 for the follow-up.

Reply:
Ok

Reply:
What filed #2459 means ?

Reply:
I created the issue #2459 to track the future improvement after your PR is merged.